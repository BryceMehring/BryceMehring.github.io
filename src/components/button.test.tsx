import React from 'react';
import Button from './button';
import { mount } from 'enzyme';

describe('button', () => {
  it('matches snapshot', () => {
    const b = mount(<Button text="Hello" link="test" />);
    expect(b.html()).toMatchSnapshot();
  });
});
