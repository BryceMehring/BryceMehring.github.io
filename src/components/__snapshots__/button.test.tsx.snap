// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button matches snapshot 1`] = `
<a
  className="btn btn-primary m-3"
  href="test"
>
  Hello
</a>
`;
